core/templates/dev/head/css/oppia.css

```diff
@@ -68,9 +68,19 @@ button.md-button.md-default-theme.oppia-learner-continue-button:hover {
 .material-icons.md-18 {
   font-size: 18px;
 }
+.material-icons.md-100 { 
+  font-size: 100px; 
+}
+.material-icons.md-40 { 
+  font-size: 40px; 
+}
 .material-icons.md-dark {
   color: rgba(0, 0, 0, 0.54);
 }
+.material-icons.menu {
+color:#fff;
```

Comment:
Please follow existing style: 2-space indentation, single space after colon, add semicolon at end of all lines